ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -23,13 +23,16 @@ import {ResourceDetails} from '../resource-details/resource-details';
 import * as AppUtils from '../utils';
 import {ApplicationResourceList} from './application-resource-list';
 import {Filters} from './application-resource-filter';
-import {urlPattern} from '../utils';
+import {nodeKey, urlPattern} from '../utils';
+import {ResourceStatus} from '../../../shared/models';
 
 require('./application-details.scss');
 
 interface ApplicationDetailsState {
     page: number;
     revision?: string;
+    showGroupedNodesPanel?: boolean;
```

Comment:
Could you just use `groupedResources` only and get rid of showGroupedNodesPanel?  The sliding panel will be opened if `groupedResources` is not empty.